users/views.py

```diff
@@ -4,6 +4,23 @@
 from rest_framework import permissions
 from users.permissions import IsAdminUser, IsOwnerOrAdmin
 
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework.reverse import reverse
+
+
+@api_view(['GET'])
```

Comment:
Same like actions: should be enum here.
But `GET` method is accepted by default so just delete it.

Reply:
Done